virttest/utils_v2v.py

```diff
@@ -554,13 +554,17 @@ def is_net_virtio(self):
             logging.error("Fail to find virtio driver")
         return False
 
-    def is_disk_virtio(self, disk="/dev/vda"):
+    def is_disk_virtio(self):
         """
         Check whether disk is virtio.
         """
-        cmd = "fdisk -l %s" % disk
+        cmd = "fdisk -l"
+        virtio_disks = r'/dev/vd[a-z]+[0-9]*'
```

Comment:
virtio-scsi can have /dev/sdxx name, this is against what you assume virtio-disks

Reply:
Thanks for pointing that. I spent some time figuring out how v2v convert disks for linux guests.
In v2v copying mode(Normal v2v convertion), v2v will auto detect if kernel supports virtio_blk, if yes, it will always convert all disks to virtio_blk for linux guests.  And this checking is only for v2v, we don't need to think about virtio_scsi here right now unless v2v changes some day.